roxctl/image/scan/scan.go

```diff
@@ -132,6 +132,7 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	c.Flags().BoolVarP(&imageScanCmd.includeSnoozed, "include-snoozed", "a", false, "the --include-snoozed flag returns both snoozed and unsnoozed CVEs if set")
 	c.Flags().IntVarP(&imageScanCmd.retryDelay, "retry-delay", "d", 3, "set time to wait between retries in seconds")
 	c.Flags().IntVarP(&imageScanCmd.retryCount, "retries", "r", 3, "Number of retries before exiting as error")
+	c.Flags().StringVarP(&imageScanCmd.cluster, "cluster", "", "", "cluster name or ID to delegate image scan to")
```

Comment:
Same as mentioned above (module here, we can leave the description since it's only image scans).

Reply:
https://github.com/stackrox/stackrox/pull/7758/commits/4526051b51195713b6bb99118bf582415a993046